### PR TITLE
feat(mapping): add OurZora.com

### PR DESCRIPTION
**Pull request by developerfred**
### OurZora Contracts 
https://github.com/developerfred/ourzora-core/blob/master/addresses/1.json

### CODE REVIEW

File: abis/OurZoraMarket.json [hunk 1]
```diff
@@ -0,0 +1 @@
...","type":"tuple"},{"internalType":"address","name":"spender","type":"address"}],"name":"setBid","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"uint256","name":"tokenId","type":"uint256"},{"components":[{"components":[{"internalType":"uint256","name":"value","type":"uint256"}],"internalType":"struct Decimal.D256","name":"prevOwner","type":"tuple"},{"components":[{"internalType":"uint256","name":"value","type":"uint256"}],"internalType":"struct Decimal.D256","name":"creator","type":"tuple"},{"components":[{"internalType":"uint256","name":"value","type":"uint256"}],"internalType":"struct Decimal.D256","name":"owner","type":"tuple"}],"internalType":"struct IMarket.BidShares","name":"bidShares","type":"tuple"}],"name":"setBidShares","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"components":[{"internalType":"uint256","name":"value","type":"uint256"}],"internalType":"struct Decimal.D256","name":"sharePercentage","type":"tuple"},{"internalType":"uint256","name":"amount","type":"uint256"}],"name":"splitShare","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"pure","type":"function"}]
```
Comment (frostiq) [inline abis/OurZoraMarket.json]: Do we really need this ABI? Seem like it's not used anywhere for now

File: src/mapping/OurZora.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  let address = event.address.toHexString();
+  if (NftContract.load(address) == null) {
+    let nftContract = new NftContract(address);
+    nftContract.name = fetchName(event.address);
+    nftContract.symbol = fetchSymbol(event.address);
+    nftContract.platform = "OurZora";
+    nftContract.save();
+  }
+
+  handleTransfer(event);
```
Comment (frostiq) [inline src/mapping/OurZora.ts]: In Zora, there are two distinct URI fields: tokenURI and metadataURI:
https://zora.engineering/protocol/smart-contracts

> This protocol deviates from the ERC-721 in that the tokenURI does not point to a valid ERC721 Metadata JSON Schema as defined in the EIP. In order to support integrity checks when updating the tokenURIs, the content and metadata of a piece of media are split into tokenURI and metadataURI, respectively.

We can leave URI mapping as it is for now, but in the future we will need to rethink how we handle URI across different protocols

**Review by frostiq — CHANGES_REQUESTED** [review]
Please merge master to this branch and resolve conflicts

**Comment by frostiq** [comment]
@developerfred, seems like there are some conflicts still, please take a look ⬇️

File: src/constants.ts [hunk 3]
```diff
@@ -8,7 +8,9 @@ export let SUPERRARE_V2 = Address.fromString("0xb932a70A57673d89f4acfFBE830E8ed7
 
 export let KNOW_ORIGIN = Address.fromString("0xFBeef911Dc5821886e1dda71586d90eD28174B7");
```
Comment (Tibike6) [inline src/constants.ts]: @developerfred @frostiq 
There is an error. Missing `d`. Correct contract address is `0xFBeef911Dc5821886e1dda71586d90eD28174B7d`

Comment (developerfred) [reply src/constants.ts]: Thanks @Tibike6!

Comment (frostiq) [reply src/constants.ts]: Ow, that's probably why I got an error during graph resync
Thanks for noticing @Tibike6!

**Pull request merged**